test/concentrate/vehicle_position_test.exs

```diff
@@ -319,6 +328,42 @@ defmodule Concentrate.VehiclePositionTest do
 
       assert Mergeable.merge(first, second) == expected
     end
+
+    test "merge/2 retains all timestamps" do
```

Comment:
Hmm, am I misunderstanding the purpose of this test, or should there be some actual values in `last_updated_by_source`?

Reply:
you understand correctly...I will look into this.